README.md

```diff
@@ -127,7 +127,8 @@ A collection of awesome things regarding Valve's Steam Deck.
         - PSA: All Arch User Repository apps installed via `yay` or `pacman` will be wiped on SteamOS updates
     - [Shader Cache Killer](https://github.com/scawp/Steam-Deck.Shader-Cache-Killer) - Allows you to manage Shader Caches stored in your internal storage.  Handy for 64GB owners who are running out of space on their Steam Deck.
     - File Transfer to/from Steam Deck
-        - (Easy) [Warpinator/Winpinator](https://steamdeckhq.com/tips-and-guides/warpinator-wireless-transfer-from-windows/) - dedicated file sharing application installed on the Steam Deck and Windows PC
+        - (Easy) [Warpinator/Winpinator](https://steamdeckhq.com/tips-and-guides/warpinator-wireless-transfer-from-windows/) - dedicated file sharing application installed on the Steam Deck and Windows PC\
+        - (Medium) [croc / scp for linux](https://github.com/schollz/croc) - simple file/folder sharing over the command line
```

Comment:
i checked this out -- croc actually looks REALLY neat.  i've never seen it before.

that said, i'm not sure if i feel very comfortable adding croc as a recommended way to xfer files.  it feels like a lot of effort to install and uses an external relay (by default).  at this point setting up sshd and using scp feels like a more established way than a single purpose tool.

i struggled with whether or not to add scp as an option here though...still food for thought.